packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.service.ts

```diff
@@ -447,17 +430,37 @@ export class ObjectMetadataService extends TypeOrmQueryService<ObjectMetadataEnt
         objectMetadataForUpdate.workspaceId,
       );
 
-      await this.objectMetadataMigrationService.createStandardRelationsUpdatesMigrations(
+      const relationsAndForeignKeysMetadata =
+        await this.objectMetadataRelationService.updateRelationsAndForeignKeysMetadata(
```

Comment:
this is only called if the object nameSingular changes (cf if (newTargetTableName !== existingTargetTableName)), while some relation fieldMetadata will have their description updated based on the label ("Rocket" -> "Rocket updated label").
Given that it is also possible to update the label without updating the name, we should maybe always call this if any name or label is updated?